code/modules/mod/modules/modules_maint.dm

```diff
@@ -384,6 +397,14 @@
 	SIGNAL_HANDLER
 	speech_args[SPEECH_SPANS] |= "upside_down"
 
+/// Prevent someone from being buckled to the wearer while atrocinator is active
+/obj/item/mod/module/atrocinator/proc/on_someone_buckled(datum/source, mob/living/buckled_mob, mob/living/buckler)
+	SIGNAL_HANDLER
+	if(buckled_mob) // Someone is being buckled to the wearer
+		balloon_alert(buckler, "they're/you're upside down!")
+		balloon_alert(mod.wearer, "atrocinator interference!")
```

Comment:
1. Rather than literally using `they're/you're`: `[buckler == mod.wearer ? "you're" : "they're"] upside down!` 
2. You don't need the second balloon alert (otherwise both will pop up at the same time and be hard to read)